### PR TITLE
Speed slower 2000

### PR DESCRIPTION
**Pull request by Rayfod**
## About The Pull Request
Makes you able to shoot the people on the fast drugs.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: code/modules/movespeed/modifiers/reagents.dm [hunk 1]
```diff
@@ -5,10 +5,10 @@
 	multiplicative_slowdown = -0.5
 
 /datum/movespeed_modifier/reagent/changelinghaste
-	multiplicative_slowdown = -2
+	multiplicative_slowdown = -0.5
```
Comment (ProbablyCarl) [inline code/modules/movespeed/modifiers/reagents.dm:8]: Not a fan of this.

**Comment by SigholtStarsong** [comment]
Document your changes in the friggin' changelog.

**Comment by Rayfod** [comment]
Draft PR

**Comment by IrkallaEpsilon** [comment]
Gotta say the "do CL" thing should be done by everyone, not just Rayford for some reason.

What this does: a speed modifier of -2 means quad speed. This is incredibly incredibly unhealthy. You can outrun bullets.
Changeling haste has that modifier. The highest easily attainable speed modifier is from stimulants in that purple syringe which has a .5, a 50% increase. This is fast but does not allow you to outrun bullets. 

A -1 modifier means double speed. Still allows you to outrun bullets but it doesnt look like you are teleporting around the map like you are doing admin stuff.

**Pull request closed without merging**